clevertap-core/src/main/java/com/clevertap/android/sdk/pushnotification/PushConstants.java

```diff
@@ -98,6 +103,17 @@ public void setRunningDevices(@XiaomiPush int runningDevices) {
             return runningDevices;
         }
 
+        public void  setServerRegion(@Nullable String region){
```

Comment:
@root-ansh-ct Either rename this to `setXPSServerRegion` or add javadoc here to explain what this method does. 
Same for the getter method